app/__init__.py

```diff
@@ -1,20 +1,9 @@
-from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-from flask_migrate import Migrate
-from flask_login import LoginManager
-from config import Config
-from flask_bootstrap import Bootstrap
 from flask_mail import Mail
+from app.app import make_app
 
 import os
 
-app = Flask(__name__)
-app.config.from_object(Config)
-db = SQLAlchemy(app)
-migrate = Migrate(app, db)
-bootstrap = Bootstrap(app)
-login = LoginManager(app)
-login.login_view = 'login'
+app, db, migrate, bootstrap, login = make_app(__name__)
```

Comment:
I pulled these out into an external file to make helpers easier. It didn't end up being that helpful (see `populate-db-with-masks.py`), but I think it's good groundwork. Also more than happy to undo it, it wasn't very hard.

Reply:
I think that we should keep it in a separate file. We eventually want to add unit tests for the exchange algorithm, and having it in a separate file might come in handy for that too.